src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -30,4 +34,24 @@ public int getQuantity() {
     public void setQuantity(int quantity) {
         this.quantity = quantity;
     }
+
+    public List<FruitTransaction> parseTransaction(String[] transactions) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+
+        for (int i = 1; i < transactions.length; i++) {
+            String transaction = transactions[i];
+            String[] parts = transaction.split(",");
+
+            Operation operation = Operation.getOperation(parts[0]);
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
+
+            FruitTransaction fruitTransaction = new FruitTransaction(operation, fruit, quantity);
+
+            fruitTransactions.add(fruitTransaction);
+        }
+        return fruitTransactions;
+    }
```

Comment:
this class shouldn't contain this logic. You have to parse transactions in a separate class